### PR TITLE
AB mount for standard dial indicator

### PR DESCRIPTION
**Pull request by teookie**
* [x ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x ] Submitted STL files are oriented and scaled properly for printing.
  * [x ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Mount for attaching standard dial indicator (not a test indicator) to AB.

#### How was it tested? 
Designed and tested by me. Used to level x rails and create manual bed mesh.

### CODE REVIEW

**Review by KiloQubit — COMMENTED** [review]
Thanks for your mod!

There's already a dial indicator mount here https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/dti-mount , and we try to avoid duplicate mods.  Is there some advantage to your mod over the existing one?

**Comment by teookie** [comment]
Fits a different style of dial indicator. Existing design is for test
indicators. This one is for standard dial indicator.

-Trent

**Comment by jurisv** [comment]
Yes, totally different beast. Bring it on! :)

File: printer_mods/teookie/AB_dial_indicator_mount/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Requires:
+- 2x M3x10mm SHCS
+- 1x M5x20mm SHCS or BHCS
+- 1x M5 nut
+
+For use with common dial indicators. Confirmed compatible with this one:
+https://www.grizzly.com/products/grizzly-magnetic-base-dial-indicator-combo/g9849?gclid=Cj0KCQjwi7yCBhDJARIsAMWFScMalpVMwU7VGmgg-EX7v9XqT1Eh7hUQgv2Z1UeLJ0F9gKKqIfKyRJwaAmnLEALw_wcB
+
+Usefull for:
```
Comment (KiloQubit) [inline printer_mods/teookie/AB_dial_indicator_mount/readme.md:13]: Please change `Usefull` to `Useful`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your reply.  Everything looks good except one type in the README noted below.

File: printer_mods/teookie/AB_dial_indicator_mount/readme.md (hunk 1 above)
Comment (teookie) [reply printer_mods/teookie/AB_dial_indicator_mount/readme.md]: Typo fixed. Thanks.

**Review by KiloQubit — APPROVED** [review]
Thanks!

**Pull request merged**